pkg/operator/ceph/object/objectstore.go

```diff
@@ -736,7 +737,13 @@ func CreatePools(context *Context, clusterSpec *cephv1.ClusterSpec, metadataPool
 	}
 
 	// get the default PG count for rgw metadata pools
-	metadataPoolPGs, err := config.GetMonStore(context.Context, context.clusterInfo).Get("mon.", "rgw_rados_pool_pg_num_min")
+	var err error
+	var metadataPoolPGs string
+	if context.clusterInfo.CephVersion.IsAtLeast(cephvers.Quincy) {
+		metadataPoolPGs, err = config.GetMonStore(context.Context, context.clusterInfo).Get("mon.", "pg_num_min")
```

Comment:
```suggestion
		metadataPoolPGs, err =  AddSettingsToMonDB(config,context.Context, context.clusterInfo).Get("mon.", "pg_num_min")
```

@thotz instead of using `.GetMonStore` method we introduced this https://github.com/rook/rook/blob/dbb98224bcef0a542d1df18c813e199eb16a01d3/pkg/operator/ceph/cluster/cluster.go#L603 where we don't need to call `GetMonStore` multiple times and run the ceph commands.  ` addSettingsToMonDB` will require small change like importing this method globally and removing receiver function.  But I think it will good

Reply:
@subhamkrai I don't think `addSettingsToMonDB()` applies in this case. A single setting is just being retrieved from the mon store, and no updates are being applied here.

Reply:
@thotz @cbodley Instead of setting this in the mon store, does the operator need to call this command? And do we just set 8 which was the default in the past, or how does rook know the recommended default?
```
ceph osd pool set <poolName> pg_num_min 8
```

Reply:
@travisn As far as I understand object-store controller is fetching the `pg_num_min` from the existing ceph cluster not setting the new value. So I don't understand setting `pg_num_min 8` for meta data pools.
Are u suggesting something like https://github.com/rook/rook/blob/master/pkg/operator/ceph/object/objectstore.go#L799 for meta data pools?

Reply:
`pg_num_min` doesn't exist in the mon store. Instead, it's a pool setting. 

The current code before this PR will do the following:
- Get the `rgw_rados_pool_pg_num_min` from the monstore
- Set that number of PGs on the pool `pg_num_min` as in the code that you linked

Now if the `rgw_rados_pool_pg_num_min` is no longer available, how does RGW team recommend to know the value to set `pg_num_min` on the rgw metadata pools? Is there no longer a recommendation? So my question is if we should just always set it to 8 since that was the default in the past? We can't leave it as the default will raise the PG count to 32, which then causes issues in the cluster with high PG count.